script.js

```diff
@@ -1,4 +1,100 @@
+//Global variables
+const GAME_STATE_ROLL_DICE = "game state roll dice";
+const GAME_STATE_CHOOSE_DICE_ORDER = "game state choose order of dice";
+const GAME_STATE_COMPARE_PLAYER_NUMBERS = "game state compare players numbers";
```

Comment:
Great job incorporating the advice from SPS and using `const` and SCREAMING_SNAKE_CASE